about.html

```diff
@@ -0,0 +1,106 @@
+<<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1" </meta>
+  <title>Manta Ray ID Project</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Nunito:200,400,600" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-modal/0.9.1/jquery.modal.min.css" />
+</head>
+
+<body>
+
+  <div class="wrapper">
+    <nav>
+      <div class="container-lg">
+        <div class="logo">
+          <a href="https://www.mantapacific.org/">
+            <img src="images/logo-manta-ray.png" alt="Manta Pacific Research Foundation Logo">
+          </a>
+          <h2>Manta Ray Identification Project</h2>
+        </div>
+        <div class="nav-options">
+          <a href="#sighting" rel="modal:open">
+            <h3>Report a Sighting</h3>
+          </a>
+          <a href="index.html">
+            <h3>Catalog</h3>
+          </a>
+          <a href="#">
+            <h3 class="active">About</h3>
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <div id="sighting" class="modal" style="background-color:var(--color-background)">
+      <div class="detail-window" style="text-align: center;">
+        <img src="images/logo-manta-ray.png" alt="Manta Pacific Research Foundation Logo">
+        <h1>Report a sighting</h1>
+        <br />
+        <p>
+          This functionality is under development.
+        </p>
+      </div>
+    </div>
+    
+    <div class="title-bar">
+        <div class="container-lg">
+          <h1>About the Kona Manta Ray Sighting Data</h1>
+        </div>
+    </div>
+
+    <div class="detail container-lg">
+    <center>
```

Comment:
Center-aligned or left-aligned is up to you. But I think since it's no longer a pop up, having it left-aligned looks better to me. Maybe just center align the "Website Created" section, not the about info?